### PR TITLE
fix minor typo in ipa-adtrust-install.1

### PR DESCRIPTION
**Pull request by realsobek**
ipa-4-5 and ipa-4-4 are affected too. I ignore branches that are not listed at http://www.freeipa.org/page/Roadmap#Current_Efforts
As far as I can see I cannot put the changes in here. Do I have to create a pull request for each one?

### CODE REVIEW

File: install/tools/man/ipa-adtrust-install.1 [hunk 1]
```diff
@@ -26,7 +26,7 @@ Adds all necessary objects and configuration to allow an IPA server to create a
 trust to an Active Directory domain. This requires that the IPA server is
 already installed and configured.
 
-Please note you will not be able to estabilish an trust to an Active Directory
+Please note you will not be able to establish an trust to an Active Directory
```
Comment (stlaz) [inline install/tools/man/ipa-adtrust-install.1]: Could you please also fix the typo "an trust" to "a trust"? Thank you :)

**Comment by realsobek** [comment]
fix has been added

**Comment by stlaz** [comment]
Thanks, ACK.

**Comment by stlaz** [comment]
Ah, I did not notice you made a second commit for this. Please, squash them.

**Comment by realsobek** [comment]
It was my fault. I knew how to squash two unpushed commits, but squashing two pushed commits was beyond me.
Thanks for your comment in #716 I had a look again and I managed it this time. :)
I squashed the last two commits (see `git log`) in my branch with `git rebase --interactive HEAD~2` and pushed the result to my fork with `git push origin master --force`

**Comment by stlaz** [comment]
Thanks, now this is ready to be pushed :)

**Comment by MartinBasti** [comment]
master:

* 298f725e5b2f820369c337dd1ab4bfd9ad3cd01f fix minor typos in ipa-adtrust-install.1

**Pull request closed without merging**